### DISCUSSION TITLE
GitHub benefits ! "benefits will be available soon"

### DISCUSSION THREAD

**Discussion by GaganHonor** [community/community#101768] [GitHub Education]
### Hi everyone,

So I applied for student developer pack and it's more than 24 hours the status window is showing 👉

![image](https://github.com/community/community/assets/141204441/723476b8-01dd-495a-9e35-d14fc2999dbb)

>So How longer is wait ?

Thanks a lot for your valuable time reading this ❤️

**Comment by kaushal878** [comment]
you have to wait 2-3 days then open GitHub page there shows welcome notification
